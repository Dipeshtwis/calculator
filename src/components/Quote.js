const Quote = () => (
    <div className = "root-home">
      <h1>Get your favourite Quote here!</h1>
      <p>Mathematics is not about numbers, equations, computations,
       or algorithms: it is about understanding. â€“William Paul Thurston</p>
    </div>
);

export default Quote;