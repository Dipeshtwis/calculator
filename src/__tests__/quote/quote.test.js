import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../../components/Quote';

it('renders correctly', () => {
  const quote = renderer.create(<Quote />).toJSON();
  expect(quote).toMatchSnapshot();
});