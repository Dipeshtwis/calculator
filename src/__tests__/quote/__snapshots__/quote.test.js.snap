// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="root-home"
>
  <h1>
    Get your favourite Quote here!
  </h1>
  <p
    className="para"
  >
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
  </p>
</div>
`;
